<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestureView</class>
 <widget class="QMainWindow" name="GestureView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestureHome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Write Com</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="writeButton">
        <property name="text">
         <string>Write Data</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Read Com</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="setButton">
        <property name="text">
         <string>&amp;Settings</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" rowspan="4">
       <widget class="QGroupBox" name="settingBox">
        <property name="title">
         <string>More Setting</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port Status</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="openButton">
           <property name="text">
            <string>open</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>PortName</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="portNamecomboBox">
           <item>
            <property name="text">
             <string>ttyPCH0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ttyPCH1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ttyUSB0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ttyUSB1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ttyUSB2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>BaudRate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="baudRatecomboBox">
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>settingBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
